api/ocm/extensions/download/handlers/ocirepo/handler.go

```diff
@@ -113,23 +124,23 @@ func (h *handler) Download(p common.Printer, racc cpi.ResourceAccess, path strin
 	}
 	log.Debug("using artifact spec", "spec", artspec.String())
 	if artspec.Digest != nil {
-		return true, "", fmt.Errorf("digest no possible for target")
+		return true, "", fmt.Errorf("digest not possible for target")
```

Comment:
could you quickly explain to me why a digest isnt possible? because you cant upload by digest?

Reply:
the digest is defined by the provided content, it is not given by the upload target spec.